tests/unit/test_config_aliases.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from packit.config.aliases import get_versions, ALIASES, get_build_targets, get_branches
+
+
+@pytest.mark.parametrize(
+    "name,versions",
+    [
+        ("fedora-29", ["fedora-29"]),
+        ("epel-8", ["epel-8"]),
+        ("fedora-rawhide", ["fedora-rawhide"]),
+        ("openmandriva-rolling", ["openmandriva-rolling"]),
+        ("opensuse-leap-15.0", ["opensuse-leap-15.0"]),
+        ("fedora-stable", ALIASES["fedora-stable"]),
+        ("fedora-development", ALIASES["fedora-development"]),
```

Comment:
I am not sure if we want to manually change also the tests or make it like this.